src/node.cc

```diff
@@ -2935,6 +2942,12 @@ static void PrintHelp() {
          "  -p, --print          evaluate script and print result\n"
          "  -i, --interactive    always enter the REPL even if stdin\n"
          "                       does not appear to be a terminal\n"
+#ifdef _WIN32
+         "  -r, --require        ';'-separated list of modules to"
```

Comment:
I was following what we do for `NODE_PATH`. The other alternative would be to use a comma, but I think that's even more likely to be in a package name. I'm open to suggestions.